wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md

```diff
@@ -0,0 +1,56 @@
+_**Main page:  [Beatmap Nominator Rules](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators/Rules)**_
```

Comment:
```diff
- _**Main page:  [Beatmap Nominator Rules](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators/Rules)**_
+ # Beatmap Veto
+ 
+ *Main page:  [Beatmap Nominator/Rules](/wiki/People/Beatmap_Nominators/Rules).*
```
I think main page would refer to Beatmap Nominators, since that is one page above this one.
Are you sure you want to link to `Beatmap Nominator/Rules` and not `Beatmap Nominator`? If so, please change `Main page` to `See also`.

Reply:
I guess this page is also of interest to mappers who want to know what a veto on their map means, so sure let's make that the parent page.

Should BN Rules be included under "See also" later on or at the end of the article, perhaps? Or is including a link in text as I've done currently enough?

Reply:
> Should BN Rules be included under "See also" later on or at the end of the article, perhaps? Or is including a link in text as I've done currently enough?

If you think it's an item of interest, sure, include BN/Rules underneath as See also.